lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -124,70 +144,66 @@ class LowerXMRPass : public LowerXMRBase<LowerXMRPass> {
           refPortsToRemoveMap[module].push_back(portNum);
     }
 
-    // Now erase all the Ops and ports of RefType.
-    // This needs to be done as the last step to ensure uses are erased before
-    // the def is erased.
-    for (auto op : llvm::reverse(opsToRemove))
-      op->erase();
-    for (auto iter : refPortsToRemoveMap)
-      if (auto mod = dyn_cast<FModuleOp>(iter.getFirst()))
-        mod.erasePorts(iter.getSecond());
-      else if (auto inst = dyn_cast<InstanceOp>(iter.getFirst())) {
-        ImplicitLocOpBuilder b(inst.getLoc(), inst);
-        inst.erasePorts(b, iter.getSecond());
-        inst.erase();
+    LLVM_DEBUG({
+      for (auto I = dataFlowClasses.begin(), E = dataFlowClasses.end(); I != E;
+           ++I) { // Iterate over all of the equivalence sets.
+        if (!I->isLeader())
+          continue; // Ignore non-leader sets.
+        for (auto MI = dataFlowClasses.member_begin(I);
+             MI != dataFlowClasses.member_end();
+             ++MI)                    // Loop over members in this set.
+          llvm::dbgs() << *MI << " "; // Print member.
+        llvm::dbgs() << "\n dataflow at leader::" << I->getData() << "\n =>";
+        if (dataflowAt.find(I->getData()) != dataflowAt.end()) {
+          for (auto init = refSendPathList[dataflowAt[I->getData()]];
+               init.second; init = refSendPathList[init.second.value()])
+            llvm::dbgs() << "\n path ::" << init.first << "::" << init.second;
+        }
+        llvm::dbgs() << "\n Done"; // Finish set.
```

Comment:
```suggestion
        llvm::dbgs() << "\n Done\n"; // Finish set.
```
(probably want a newline at least between this and the next iteration's printing of the EC members)